frame/App/Body/Main/Layout/components/AddItem.js

```diff
@@ -57,6 +71,14 @@ const AddItem = ({
       >
         Add New Item
       </button>
+      <button
+        type="button"
+        disabled={value.length === 0}
+        onClick={markPriority}
+      >
+        {priority ? <Icon icon="flag-variant" /> : <Icon icon="flag-variant-outline" />}
```

Comment:
This is much simpler don't you think?
```suggestion
        <Icon icon={priority ? 'flag-variant' : 'flag-variant-outline' />
```